technical-reports/README.md

```diff
@@ -0,0 +1,58 @@
+# DTCG Technical Reports
+
+This directory contains the source code for the [Design Token Community Group's (DTCG) technical reports](https://design-tokens.github.io/community-group/).
+
+<!-- TOC depthfrom:2 -->
+
+- [Pre-requisites](#pre-requisites)
+- [Initial setup](#initial-setup)
+- [Local previews](#local-previews)
+- [Editing](#editing)
+- [Deployments](#deployments)
+
+<!-- /TOC -->
+
+## Pre-requisites
+
+In order to preview edits locally, you will need the following installed on your machine:
+
+- [Node.js](https://nodejs.org/en/) >= 16.8
+  - If you use [`nvm`](https://github.com/nvm-sh/nvm), you can run `nvm install` to install and use the recommended version of Node.
+- [Yarn](https://yarnpkg.com/)
+
+## Initial setup
+
+After cloning this repo, you need to install the dependencies:
+
+```
+yarn
```

Comment:
I switched the repo to npm yesterday, I should have given you a heads up, sorry!

Reply:
No worries. (I was meaning to ask if we really need `yarn` here anyway, so glad to hear you've made that change)